src/components/accessibility/screen_reader_only/screen_reader_only.styles.ts

```diff
@@ -8,13 +8,29 @@
 
 import { css } from '@emotion/react';
 
-export const euiScreenReaderOnlyStyles = () => {
-  return css`
+/*
+ * Mixin
+ */
+export const euiScreenReaderOnly = () => `
     position: absolute;
     left: -10000px;
     top: auto;
     width: 1px;
     height: 1px;
     overflow: hidden;
```

Comment:
This doesn't match what's currently in our SCSS, and it's important that we do for mutliple cross browser reasons that we 100% should leave the comments in for

```suggestion
  // Take the element out of the layout
  position: absolute;
  // Keep it vertically inline
  top: auto;
  // Chrome requires a left value, and Selenium (used by Kibana's FTR) requires an off-screen position for its .getVisibleText() to not register SR-only text
  left: -10000px;
  // The element must have a size (for some screen readers)
  width: 1px;
  height: 1px;
  // But reduce the visible size to nothing
  clip: rect(0 0 0 0);
  clip-path: inset(50%);
  // And ensure no overflows occur
  overflow: hidden;
  // Chrome requires the negative margin to not cause overflows of parent containers
  margin: -1px;
```